INSTALL.md

```diff
@@ -51,7 +51,7 @@ DockerHub allows only one private repository per free account. In case the Docke
 
 ### Kpack
 
-[Kpack](https://github.com/pivotal/kpack) is used to build runnable applications from source code using [Cloud Native Buildpacks](https://buildpacks.io/). Follow the [instructions](https://github.com/pivotal/kpack/releases/latest) to install the latest version.
+[Kpack](https://github.com/pivotal/kpack) is used to build runnable applications from source code using [Cloud Native Buildpacks](https://buildpacks.io/). Follow the [instructions](https://github.com/pivotal/kpack/blob/main/docs/install.md) to install the latest version.
```

Comment:
Hey @kei-yamazaki thank you for your contribution. We generally advise our users to use the latest released version of our dependencies such as `kpack`. That's why we are pointing to the release, rather than the tip of main. Maybe you can incorporate this by keeping the link to the release. Something like:

Follow the [instructions](https://github.com/pivotal/kpack/blob/main/docs/install.md) to install the [latest release](https://github.com/pivotal/kpack/releases/latest).